spec/controllers/api/v1/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'when creating a valid book suggestion' do
+      let(:valid_params) do
+        {
+          book_suggestion: {
```

Comment:
You can use `attributes_for` from factory bot here

Reply:
A much better approach